draft-ietf-oauth-status-list.md

```diff
@@ -167,9 +169,13 @@ The following rules apply to validating the "status" (status) claim
 
 1. The claim value MUST be a valid JSON object.
 
-2. The claim value object MUST contain an "idx" (index) member with a numeric value that represents the index to check for status information in the Status List for the current JWT. The value of this member MUST be a non-negative number, containing a value of zero or greater.
+2. The claim value object MUST contain a member called "status_list" that signals that status checks for this token can be done using the status list mechanism defined in this document. For the "status_list" object, the following rules apply:
```

Comment:
If we make this a MUST then won't other future mechanisms only be additional rather than alternatives?

Reply:
this is only a MUST for the mechanism used in the specification. If Giuseppe wants to implement an OCSP-like mechanism than it would look mlike:
```json
"status" : {
  "new-mechanism-name" : {
    "whatever-parameter" : "whatever-value"
  }
}  
```

Reply:
I think we may need some text to the introduction that explains this in more detail. However its more better to merge #89 first and then start working from there